src/ucp/core/ucp_request.h

```diff
@@ -53,10 +53,11 @@ enum {
     UCP_REQUEST_FLAG_RNDV_FRAG            = UCS_BIT(15),
     UCP_REQUEST_FLAG_RECV_AM              = UCS_BIT(16),
     UCP_REQUEST_FLAG_RECV_TAG             = UCS_BIT(17),
+    UCP_REQUEST_FLAG_RMA_SW_SENT          = UCS_BIT(18),
```

Comment:
can we check by uct.func field, instead of adding flag?

Reply:
done